compiled_starters/cpp/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.13)
+
 project(bittorrent-starter-cpp)
-set(CMAKE_CXX_STANDARD 20) # Enable the C++20 standard
+
 file(GLOB_RECURSE SOURCE_FILES src/*.cpp src/lib/*.hpp)
```

Comment:
@ryan-gang let's change this to `src/*.hpp`, don't think we need to assume a user will only put stuff into `src/lib` (unless you know of a good reason for this?)